src/deps.ts

```diff
@@ -378,5 +382,5 @@ export interface AutoEncrypter {
   teardown(force: boolean, callback: Callback): void;
   encrypt(ns: string, cmd: Document, options: any, callback: Callback<Document>): void;
   decrypt(cmd: Document, options: any, callback: Callback<Document>): void;
-  readonly csfleVersionInfo: { version: bigint; versionStr: string } | null;
+  readonly cryptSharedLibVersionInfo: { version: bigint; versionStr: string } | null;
```

Comment:
Would we also want to mark this property with a tag?

Reply:
Yeah, that makes sense. I think this is also the only `AutoEncrypter` property that you might still want to expose as public API in general.